api/v1alpha1/odfinfoconfig.go

```diff
@@ -31,6 +33,7 @@ type InfoStorageCluster struct {
 	StorageProviderEndpoint string               `yaml:"storageProviderEndpoint"`
```

Comment:
Should this be removed?

Reply:
Let's keeps this for now until we are sure no one else (including Fusion) is using this field.